radis/lbl/loader.py

```diff
@@ -496,6 +497,61 @@ def __init__(self):
         self.total_lines = 0  #: int : number of lines in database.
 
 
+class CalcTime(object):
```

Comment:
Detail  : the class itself could be moved outside of lbl/loader.py ; maybe in radis/misc/debug.py or a new radis/misc/profiler.py ?

Reply:
Indeed. A new file will be better!